packages/design-picker/src/components/index.tsx

```diff
@@ -102,9 +104,80 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 	);
 };
 
+interface DesignButtonCoverProps {
+	design: Design;
+	onSelect: ( design: Design ) => void;
+	onPreview: ( design: Design ) => void;
+}
+
+const DesignButtonCover: React.FC< DesignButtonCoverProps > = ( {
+	design,
+	onSelect,
+	onPreview,
+} ) => {
+	const { __ } = useI18n();
+	const isBlankCanvas = isBlankCanvasDesign( design );
+	const designTitle = isBlankCanvas ? __( 'Blank Canvas' ) : design.title;
+
+	return (
+		<div className="design-button-cover">
+			<div className="design-button-cover__button-groups">
+				<Button
+					className="design-button-cover__button"
+					isPrimary
+					onClick={ () => onSelect( design ) }
+				>
+					{
+						// translators: %s is the title of design with currency. Eg: Alves
+						sprintf( __( 'Start with %s', __i18n_text_domain__ ), designTitle )
+					}
+				</Button>
+				<Button className="design-button-cover__button" onClick={ () => onPreview( design ) }>
+					{
+						// translators: %s is the title of design with currency. Eg: Alves
+						sprintf( __( 'Preview %s', __i18n_text_domain__ ), designTitle )
+					}
+				</Button>
+			</div>
+		</div>
+	);
+};
+
+interface DesignButtonContainerProps extends DesignButtonProps {
+	onPreview?: ( design: Design ) => void;
+}
+
+const DesignButtonContainer: React.FC< DesignButtonContainerProps > = ( {
+	onPreview,
+	...props
+} ) => {
+	const isDesktop = useViewportMatch( 'large' );
```

Comment:
here too I think we should be using `useDesktopBreakpoint` from `viewport-react`